app/locales/cn.ts

```diff
@@ -365,8 +365,9 @@ const cn = {
       History: (content: string) => "这是历史聊天总结作为前情提要：" + content,
       Topic:
         "使用四到五个字直接返回这句话的简要主题，不要解释、不要标点、不要语气词、不要多余文本，不要加粗，如果没有主题，请直接返回“闲聊”",
-      Summarize:
-        "简要总结一下对话内容，用作后续的上下文提示 prompt，控制在 200 字以内",
+      Summarize: // 由H0llyW00dzZ改进 参考：https://github.com/H0llyW00dzZ/GoGenAI-Terminal-Chat
```

Comment:
Thank you for your contribution, but it is not recommended to add promotional content in the comments alone.

Reply:
You can remove it, it doesn't mean promote aswell